app/Game.py

```diff
@@ -0,0 +1,125 @@
+from typing import Dict
```

Comment:
The import statement `from typing import Dict` is correct, but it is not used in the code. If you don't plan to use `Dict`, consider removing this import to clean up the code.